lib/ripper-tags/data_reader.rb

```diff
@@ -81,8 +81,11 @@ def clean_path(file)
 
     def each_file(&block)
       return to_enum(__method__) unless block_given?
-      options.files.each do |file|
-        resolve_file(file, &block)
+      if (in_file = options.input_file)
```

Comment:
If we were in a loop I would disagree, because it's much faster than going through the OpenStruct again, but in this case it just saved some typing.

👍